src/meta/meta_ckpt.c

```diff
@@ -934,11 +934,18 @@ __wt_meta_sysinfo_set(WT_SESSION_IMPL *session)
     WT_DECL_RET;
     WT_TXN_GLOBAL *txn_global;
     wt_timestamp_t oldest_timestamp;
+    uint32_t i;
+    char buf2[256];
```

Comment:
Instead of `256` magic number, there should be some macro something like below WT_TS_HEX_STRING_SIZE.

Reply:
Fixed.